source/utilities.tex

```diff
@@ -6705,6 +6712,16 @@
     shared_ptr<T> allocate_shared(const A& a, size_t N,
                                   const remove_extent_t<T>& u);                 // \tcode{T} is \tcode{U[]}
 
+  template<class T>
+    shared_ptr<T> make_shared_default_init();                                   // \tcode{T} is not \tcode{U[]}
```

Comment:
This entire block of changes needs to move further down, according to the ingress paper.

Reply:
yup, thanks!